client/main.lua

```diff
@@ -66,33 +67,35 @@ local function getVehicle()
     return vehicle
 end
 
----manages the opening of locks
----@param vehicle number? The entity number of the vehicle.
----@param state boolean? State of the vehicle lock.
----@param anim any Aniation
+--- Manages the opening of locks
+--- @param vehicle number? The entity number of the vehicle.
```

Comment:
To be consistent with the project remove the white character between “---” and “@param”